src/httpserver.cpp

```diff
@@ -149,10 +151,68 @@ static GlobalMutex g_httppathhandlers_mutex;
 static std::vector<HTTPPathHandler> pathHandlers GUARDED_BY(g_httppathhandlers_mutex);
 //! Bound listening sockets
 static std::vector<evhttp_bound_socket *> boundSockets;
+
+/**
+ * @brief Helps keep track of open `evhttp_connection`s with active `evhttp_requests`
```

Comment:
tiny nit (if you have to retouch):
```suggestion
 * @brief Helps keep track of open `evhttp_connection`s with active `evhttp_request`s
```